filesystem/src/main/java/org/frankframework/filesystem/sftp/SftpFileRef.java

```diff
@@ -78,7 +78,10 @@ private void setFolder(String folder) {
 	/** Returns the canonical name inclusive file path when present */
 	public String getName() {
 		String prefix = folder != null ? folder + "/" : "";
-		return prefix + name;
+		if (org.springframework.util.StringUtils.hasLength(prefix) || name != null) {
```

Comment:
Graag dezelfde API blijven gebruiken als we overal al gebruiken:

```suggestion
		if (StringUtils.isNotEmpty(prefix) || name != null) {
```